packages/mdc-menu/simple/foundation.js

```diff
@@ -149,6 +165,20 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
     this.adapter_.deregisterBodyClickHandler(this.documentClickHandler_);
   }
 
+  /**
+   * @param {Corner} corner Default anchor corner alignment of top-left menu corner.
+   */
+  setAnchorCorner(corner) {
```

Comment:
Document this in the README

Reply:
Done

Reply:
If this comment has been addressed, I'm not sure the latest commits have been pushed to this remote branch?